sinch/domains/verification/__init__.py

```diff
@@ -156,16 +211,11 @@ def get_by_reference(self, reference) -> GetVerificationStatusByReferenceRespons
             )
         )
 
-    def get_by_id(self, id) -> GetVerificationStatusByIdResponse:
```

Comment:
I'm not sure to understand why this endpoint has been removed (moreover without any deprecation warning as mentioned by @JPPortier), it still exists in the specification: https://gitlab.com/sinch/sinch-projects/enterprise-and-messaging/documentation/developer-experience/oas-documentation/-/blob/master/services/verification-oas/verification.yaml?ref_type=heads#L105

Reply:
Oops! I've misunderstood the spec file. The endpoint is back.